template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -5,44 +5,32 @@
 }
 
 <h4>@ViewData["Title"]</h4>
-@if (Model.HasAny2faProviders)
+@if (Model.HasAny2faProviders && Model.Is2faEnabled)
```

Comment:
Basically move away from the old attempt to make it easy to add arbitrary 2fa providers, and just tailor the template for only the authenticator flow.  They could still add more 2fa providers and additional flows, they would just need to do that all themselves.